### ISSUE TITLE
Examples Of How To Use?

### ISSUE THREAD

**Issue by Soupy** [RobinHerbots/Inputmask#89]
I'm trying to use this inputmask plugin by following the examples shown, but, I simply cannot make them work. For example, if I use:

$('#test').inputmask('999', { numericInput: true });

I can type ANYTHING into that text input field!!!!  Obviously I'm doing something wrong, but, what? This is where some included examples that demonstrate this plugin's capabilities would assist me greatly. I'm referring to full markup examples.

Thanks for considering this suggestion. And, BTW, your work looks very promising.

**Comment by RobinHerbots** [comment]
Hi Soupy, 

I should create some kind of demo page with examples, but I didn't find the time todo so.

Anyway I will create it some time ;-)

Can you send your code so I can have a look, why it is not working for you.  Which version of jquery do you use?

best regards, 

Robin

**Comment by Soupy** [comment]
I'll try to get you some code to review. Basically, I'm just replacing the older version of your plugin (which worked for me) with this new one.

Should things just work as before with this new version or do I need to change the way I call it?

Some demos would be fantastic!

***\* UPDATE ****

I've discovered that if I try to use the free version of HTML-Kit to test this plugin, it does not work correctly. If, however, I take the same HTML and open it with a "real" browser, it works! FYI, here's my markup:

```
<head>
    <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js"></script>
            <script type="text/javascript" src="js/jquery.inputmask.js"></script>
    <script type="text/javascript" />
         $().ready(function() {
        $('#test').inputmask('€ 999.999.999,99', { numericInput: true } );
        $('#phone').inputmask("mask", {"mask": "(999) 999-9999"});
     });
    </script>
</head>
<body>
    <form>
        <label>Test</label>
        <input type="text" value=" " id="test" ></input>
        <label>Phone</label>
        <input type="text" value=" " id="phone" ></input>
    </form>
</body>
```

Sorry to have bothered you with this. BTW, the example code for "Set Defaults" should be "autoUnmask" (uppercase U).

**Comment by RobinHerbots** [comment]
Which version of jquery do you use?  You should use 1.7 or up.

2012/7/28 Soupy <
reply@reply.github.com

**Comment by Soupy** [comment]
I'm using jQuery version 1.7.2. As mentioned before, it appears that the free version of HTML-Kit doesn't like this plugin and it works better in an actual browser (IE, Firefox).

I'm having another difficulty trying to get numeric input (actually currency) to work for me. I'd like to have a format like "$ 99,999.99" such that when a user enters "1234", I can get an unmasked value of 1234.00. Is that possible and if so, how do I construct the parameters?

TIA

**Comment by RobinHerbots** [comment]
Hi Soupy,

I never used HTML-kit and never tested the plugin within HTML-Kit.

For the numeric input.  This is not possible.  Unmasking just strips away
all masking of the input.

You may make a new issue for it as a feature request. When I find the time
I will implement it.

best regards,

Robin

2012/7/29 Soupy <
reply@reply.github.com

**Comment by RobinHerbots** [comment]
@Soupy , 

I updated the readme with the autoUnmask instead of autounmask  ;-)

Thx

Robin
